From 34cffc07e604e668990824422be555a39f890dbe Mon Sep 17 00:00:00 2001
From: David Steele <dsteele@gmail.com>
Date: Sun, 8 Sep 2013 15:06:31 -0400
Subject: [PATCH 01/20] Establish sasl authentication in alternate auth file if
 auth not established.

The alternate auth file is currently defined to support a format
with the username on the first line, and the password on the second.
This allows it to support the more secure sasl format used by the
default auth file.

If the alternate auth file is specified and authentication fails, this
will lead to the applications prompting for credentials, which are
ultimately stored in the sasl form.
---
 cloudprint/cloudprint.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cloudprint/cloudprint.py b/cloudprint/cloudprint.py
index 88aed39..614c501 100755
--- a/cloudprint/cloudprint.py
+++ b/cloudprint/cloudprint.py
@@ -436,8 +436,8 @@ def main():
                 account_file.close()
 
             else:
-                LOGGER.info('Unable to find account file')
-                return
+                cpp.username = raw_input('Google username: ')
+                cpp.password = getpass.getpass()
         else:
           cpp.username = raw_input('Google username: ')
           cpp.password = getpass.getpass()
-- 
1.8.4.rc3

